src/sentry/incidents/models/alert_rule.py

```diff
@@ -336,14 +342,24 @@ def subscribe_projects(
         )
         # NOTE: AlertRuleMonitorType.ACTIVATED will be conditionally subscribed given activation triggers
         # On activated subscription, additional query parameters will be added to the constructed query in Snuba
+        created_subscriptions = []
         if self.monitor_type == monitor_type.value:
```

Comment:
Forgive me who has little context on this, so if the monitor type passed in is continuous, would there be subscriptions? And then if it's activated, we create activations for the subscriptions?

Reply:
Yup correct

So, default is continuous.
So if the AlertRule (self) monitor_type is continuous, then we subscribe the projects.

For activated AlertRules, we'll call this subscribe_projects when the activation condition is triggered (eg. on release creation or deploy creation)